### PR TITLE
Package with numpy wrappers for blocked matrices

### PR DESCRIPTION
**Pull request by vahtras**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/blocked-matrix-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  sha256: 550f5f7ee147b513abca70432cd85b0fe4fc87218e0447e9d538db33ec00d02d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - numpy
```
Comment (synapticarbors) [inline recipes/blocked-matrix-utils/meta.yaml]: Is `numpy` required to build the package? It doesn't look like it from the `setup.py`

File: recipes/blocked-matrix-utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - python >=3
+
+test:
+  imports:
+    - util
+
+about:
+  home: "https://github.com/vahtras/util"
+  license: LGPL-2.1
```
Comment (synapticarbors) [inline recipes/blocked-matrix-utils/meta.yaml]: The license file in the repository doesn't match and is MIT, not GPL

Comment (vahtras) [reply recipes/blocked-matrix-utils/meta.yaml]: Thanks that was a mistake. Will change.

File: recipes/blocked-matrix-utils/meta.yaml (hunk 1 above)
Comment (vahtras) [reply recipes/blocked-matrix-utils/meta.yaml]: So the host section is about build dependency? Sorry, I am a novice here and confused. No, this is pure Python, so should I remove it?

Comment (synapticarbors) [reply recipes/blocked-matrix-utils/meta.yaml]: See: https://conda-forge.org/docs/maintainer/adding_pkgs.html#requirements

**Comment by synapticarbors** [comment]
@vahtras -- Thank you for your contribution to conda-forge!

**Pull request merged**